DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,198 @@
+describe('login', () => {
+
+    beforeEach(() => {
+        cy.visit("https://demoqa.com/automation-practice-form")
+    });
+
+    it('submit successfully with a needed input', () => {
+        cy.get("#firstName").type("Phuong")
+        cy.get("#lastName").type("Vo")
+        cy.get('#userEmail').type('vnphuong3001@gmail.com')
+        //  cy.get('#dateOfBirthInput').click()
+        //  cy.get('.react-datepicker__month-select').select('January')
+        // cy.get('.react-datepicker__year-select').select('2001')
+        // cy.get('#subjectsContainer').type('m')
+        // cy.get('css-12jo7m5 subjects-auto-complete__multi-value__label').click()
+
+    
+
+        cy.get("label[for='gender-radio-1']").click();
```

Comment:
![image](https://github.com/hoanghong72/WC3-Homework/assets/147176863/be903993-addd-4321-9f43-0926931512d9)
you should catch this locator because the value attribute is more specific and easier to re-use in other cases